styles.css

```diff
@@ -1,42 +1,17 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
```

Comment:
[OPTIONAL]

- [ ] I think you're having an issue with `font-weight`s because you imported only one style for this font (the basic weight). You can fix this issue by selecting the bold styles of this font in google fonts then updating this link, your headers will look better and they will match the design on Figma 👍 

Here is the correct import URL : 
```
@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
```